packages/core/src/page.js

```diff
@@ -201,14 +120,15 @@ export default class Page extends EventEmitter {
     ) + '}';
 
     // send the call function command
-    let { result, exceptionDetails } = await this.send('Runtime.callFunctionOn', {
-      functionDeclaration: fnbody,
-      arguments: args.map(value => ({ value })),
-      executionContextId: this.contextId,
-      returnByValue: true,
-      awaitPromise: true,
-      userGesture: true
-    });
+    let { result, exceptionDetails } =
+      await this.session.send('Runtime.callFunctionOn', {
```

Comment:
_enter the bikeshed zone_

I have very loose feels on this but I liked the function call on the same line as the var 🤨 

_leave the bikeshed zone_

Reply:
On the same line causes it to extend too close to the 100 char range which typically causes overflow and scrolling unless you have a high res monitor or smaller font sizes. If we don't like this syntax on the next line, I'd propose adding a rule to the linter and I can wrap it in parenthesis like we do elsewhere